src/components/TokenBottomSheet.tsx

```diff
@@ -224,76 +230,90 @@ function TokenBottomSheet({
+        }}
+      />
+      <View style={styles.headerContainer} onLayout={handleMeasureHeader}>
+        <Text style={[styles.title, titleStyle]}>{title}</Text>
+        {searchEnabled && (
+          <SearchInput
+            placeholder={t('tokenBottomSheet.searchAssets') ?? undefined}
+            value={searchTerm}
+            onChangeText={(text) => {
+              setSearchTerm(text)
+              sendAnalytics(text)
             }}
+            style={styles.searchInput}
+            returnKeyType={'search'}
+            // disable autoCorrect and spellCheck since the search terms here
+            // are token names which autoCorrect would get in the way of. This
+            // combination also hides the keyboard suggestions bar from the top
+            // of the iOS keyboard, preserving screen real estate.
+            autoCorrect={false}
+            spellCheck={false}
+          />
+        )}
+        {filterChips.length > 0 && (
+          <FilterChipsCarousel
+            chips={filters}
+            onSelectChip={handleToggleFilterChip}
+            primaryColor={colors.successDark}
+            secondaryColor={colors.successLight}
+            style={styles.filterChipsCarouselContainer}
+            forwardedRef={filterChipsCarouselRef}
+            scrollEnabled={false}
           />
-          <View style={styles.headerContainer} onLayout={handleMeasureHeader}>
-            <Text style={[styles.title, titleStyle]}>{title}</Text>
-            {searchEnabled && (
-              <SearchInput
-                placeholder={t('tokenBottomSheet.searchAssets') ?? undefined}
-                value={searchTerm}
-                onChangeText={(text) => {
-                  setSearchTerm(text)
-                  sendAnalytics(text)
-                }}
-                style={styles.searchInput}
-                returnKeyType={'search'}
-                // disable autoCorrect and spellCheck since the search terms here
-                // are token names which autoCorrect would get in the way of. This
-                // combination also hides the keyboard suggestions bar from the top
-                // of the iOS keyboard, preserving screen real estate.
-                autoCorrect={false}
-                spellCheck={false}
-              />
-            )}
-            {filterChips.length > 0 && (
-              <FilterChipsCarousel
-                chips={filters}
-                onSelectChip={handleToggleFilterChip}
-                primaryColor={colors.successDark}
-                secondaryColor={colors.successLight}
-                style={styles.filterChipsCarouselContainer}
-                forwardedRef={filterChipsCarouselRef}
-                scrollEnabled={false}
-              />
-            )}
-          </View>
-        </View>
-      </BottomSheetBase>
+        )}
+      </View>
+    </View>
+  )
+
+  return (
+    <>
+      {isScreen ? (
+        <View style={{ height: variables.height * 0.9 }}>{content}</View>
```

Comment:
one implication of this is that the bottom sheet will always be 90% high regardless of the number of tokens. If this is not set and the token list exceeds the screen height, the bottom sheet occupies the full screen. The BottomSheetScrollView (which we use for other bottom sheet screens) specifies a maxHeight to the scroll view, but wasn't able to use that here. Open to other ideas.

Reply:
what about using just `maxHeight` here? you can remove the `flex:1` on the `container` style to make that work (might need to bring the `container` into the render method and use it only if not `isScreen`)

Reply:
this worked. thanks!